gameros/airootfs/etc/systemd/system/dbus-org.freedesktop.nm-dispatcher.service

```diff
@@ -0,0 +1 @@
+/usr/lib/systemd/system/NetworkManager-dispatcher.service
```

Comment:
What does the NetworkManager-dispatcher do?

Reply:
Have no clue. But is needed by the `NetworManager.service` unit and when you enable it with systemd it gets symlinked as well.

Reply:
Makes sense, thanks! Then this one can be closed.